src/app/Main/Body/QueryPage/Results/Charts/BarChartResults.component.tsx

```diff
@@ -0,0 +1,126 @@
+import React from "react";
+import { ApexOptions } from "apexcharts";
+import ReactApexChart from "react-apexcharts";
+import { primaryColor } from "../../../../../Shared/Style/Style-MainTheme.component";
+
+type XYType = { name: string; data: number[] };
+
+function processingJsonApex(

```

Comment:
Consider moving this conversion up to e.g. the BarChartComponent, so the props for this component become more fitted to its functionality (to display a chart).

Reply:
Done. Commit "Move processing Data to parent & change types"